cfme/intelligence/reports/reports.py

```diff
@@ -152,6 +165,18 @@ class ReportDetailsView(CloudIntelReportsView):
     @View.nested
     class report_info(WaitTab):  # noqa
         TAB_NAME = "Report Info"
+        # Keeping `group_title` empty since the summary form has no title
+        report_id = SummaryFormItem("", "ID")
+        title = SummaryFormItem("", "Title")
+        primary_filter = SummaryFormItem("", "Primary (Record) Filter")
+        secondary_filter = SummaryFormItem("", "Secondary (Display) Filter")
+        sort_by = SummaryFormItem("", "Sort By")
+        based_on = SummaryFormItem("", "Based On")
+        user = SummaryFormItem("", "User")
+        group = SummaryFormItem("", "EVM Group")
+        updated_on = SummaryFormItem("", "Updated On")
+        report_schedule_data = Table('//*[@id="report_info"]/table[1]')
+        report_widgets_data = Table('//*[@id="report_info"]/table[2]')
```

Comment:
Q: hummmmm I hate indexing in locator; can you find some relative locator for table?

Reply:
Good that you caught this one @digitronik, xpaths weren't set properly here.
I have now set xpath on the basis of value of second title head in table.